Vagrantfile

```diff
@@ -22,7 +22,7 @@ VAGRANTFILE_API_VERSION = "2"
 # General config
 enable_dns = false
 num_workers = 3
-ram_megabytes = 1280
+ram_megabytes = 300
```

Comment:
These are system tests of ducktape itself, so there's no jvm, just smallish bash commands on the remote workers